docs/pages/get_started/designers.js

```diff
@@ -44,17 +44,13 @@ In your first couple of weeks at Pinterest, you will receive an invite from your
           description={`
 **Please note: The following plugins are only available for Pinterest employees.**
```

Comment:
We might not need this is we move the links to show the private icon as per comment below

Reply:
Yeah, man, I'd _love_ to be able to use `PrivateLink` and thus get rid of this line. Maybe at some point we can update our `MainSection.Subsection` component to use MDX?